Solutions/ApacheHTTPServer/Data Connectors/Connector_ApacheHTTPServer_agent.json

```diff
@@ -2,7 +2,7 @@
     "id": "ApacheHTTPServer",
     "title": "Apache HTTP Server",
     "publisher": "Apache",
-    "descriptionMarkdown": "The Apache HTTP Server data connector provides the capability to ingest [Apache HTTP Server](http://httpd.apache.org/) events into Azure Sentinel. Refer to [Apache Logs documentation](https://httpd.apache.org/docs/2.4/logs.html) for more information.",
+    "descriptionMarkdown": "The Apache HTTP Server data connector provides the capability to ingest [Apache HTTP Server](http://httpd.apache.org/) events into Microsoft Sentinel. Refer to [Apache Logs documentation](https://httpd.apache.org/docs/2.4/logs.html) for more information.",
```

Comment:
Please verify, if we need to update this text as shown in below screenshot from the text change document under "Apache Http Server-->Data Connector" section